pkg/docker/containers-view.jsx

```diff
@@ -185,14 +209,59 @@ var ContainerList = React.createClass({
         this.setState({ containers: containers });
     },
 
+    setNewProblem: function (c_id, url, message) {
+        /* New problem is always displayed, no matter if the same problem is
+         * already shown. It is because user may be interested into dynamic
+         * watching of the problems occurring. After refreshing the site, only
+         * the latest occurrence is displayed.
+         */
+        var known_problems = this.state.problems;
+        if ( c_id in known_problems )
+            known_problems[c_id].push([url, message]);
+        else
+            known_problems[c_id] = [[url, message]];
+        this.setState({ problems: known_problems });
+    },
+
+    newProblemOccurred: function (event, problem_path) {
+        var entry = this.problems_client.proxy('org.freedesktop.Problems2.Entry', problem_path);
+        var self = this;
+        entry.wait(function() {
+            util.find_container_log(entry, self.setNewProblem);
+         });
+    },
+
     componentDidMount: function () {
+        var self = this;
+        this.problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
+        this.service = this.problems_client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
+        this.problems = this.problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
+        this.problems.wait(function() {
+            try{
+                self.service.GetSession().done(function(session_path) {
+                    var session = self.problems_client.proxy('org.freedesktop.Problems2.Session', session_path);
+                    session.wait(function() {
+                        session.Authorize({}); //ignore return code
+                    });
+                });
+            }
+            catch (err) {
+                // no ABRT present, continue
```

Comment:
Could this get a check if it's actually about missing ABRT? What if Authorize() fails?

Reply:
Authorize should not fail. It returns -1 when failed. Of course if there is some bug, it can fail unexpectedly, but we do not want cockpit users bother with it.

Reply:
If it should not fail, then why do we catch the failure here? Lets log it (javascript console) or handle the exception appropriately.

Reply:
It was because "self.service.GetSession()" is undefined when there is no ABRT. But check such as `typeof self.service.GetSession !== "undefined"` would do. I'll rewrite it to that.